ivy/functional/frontends/numpy/linear_algebra/linalg/norms_and_other_numbers.py

```diff
@@ -0,0 +1,13 @@
+# global
+import ivy
+
+# slogdet
+
+
+def slogdet(a):
+    sign, logabsdet = ivy.slogdet(a)
+    ret = ivy.concat((ivy.reshape(sign, -1), ivy.reshape(logabsdet, -1)))
```

Comment:
can you pass -1 as a tuple, because `ivy.reshape` takes a sequence of int for `shape`